deploy_tools/deploy.py

```diff
@@ -71,6 +71,40 @@ def send_function_call_transaction(
     return wait_for_successful_transaction_receipt(web3, tx_hash)
 
 
+def send_transaction(*, web3: Web3, transaction_options: Dict, private_key=None):
```

Comment:
this has a lot in common with send_function_call_transaction, did you at least try to remove some of the redundancy?

Reply:
Yes, I can to do some things like
```
if function_call: buid_transaction()
else: fill_transaction_default()
```

But then it feels like I have a function that has two different purpose depending on whether it intends to transact a function call or simply transactions options.

Otherwise I can attempt to add the data in the transaction option for `send_function_call_transaction()` and make it call `send_transaction()` but it feels unnecessarily complicated.